commit 4ea1f5e15e78cd019b46cccbfc0445b9098658c3
Author: Martin Pitt <martin.pitt@ubuntu.com>
Date:   Sun Jan 9 13:04:09 2011 -0600

    Fix gtk_icon_theme_load_icon annotation
    
    Cherrypicked from master (e0aa12eb0a) to fix return value transfer annotation.

diff --git a/gtk/gtkicontheme.c b/gtk/gtkicontheme.c
index 82636ff..23a6a94 100644
--- a/gtk/gtkicontheme.c
+++ b/gtk/gtkicontheme.c
@@ -1513,8 +1513,8 @@ gtk_icon_theme_error_quark (void)
  * returned by this function. Otherwise GTK+ may need to keep the old 
  * icon theme loaded, which would be a waste of memory.
  * 
- * Return value: the rendered icon; this may be a newly created icon
- *  or a new reference to an internal icon, so you must not modify
+ * Return value: (transfer full): the rendered icon; this may be a newly
+ *  created icon or a new reference to an internal icon, so you must not modify
  *  the icon. Use g_object_unref() to release your reference to the
  *  icon. %NULL if the icon isn't found.
  *
